web/packages/teleport/src/components/Authenticated/Authenticated.tsx

```diff
@@ -67,6 +67,7 @@ const Authenticated: React.FC<PropsWithChildren> = ({ children }) => {
         if (result.requiresDeviceTrust === TrustedDeviceRequirement.REQUIRED) {
           session.setDeviceTrustRequired();
         }
+        storageService.setLoginTime(Date.now());
```

Comment:
If I'm correct, it's not a login time but 'session start' time. We don't update on every page reload only because the check in `setLoginTime` only sets the time once. Wouldn't it be simpler if we called it after a successful login?

Reply:
Yes, however for SSO logins, we don't have any way on the frontend to keep track of the successful login, because all we do is redirect to the SSO provider which manages it and then redirects the user to the home page directly.